### ISSUE TITLE
Event logs are not consistent from 5.0.4.1 to 5.0.5.0

### ISSUE THREAD

**Issue by Jainbrt** [IBM/ibm-spectrum-scale-csi#219]
**Describe the bug**
Event logs are not consistent from 5.0.4.1 to 5.0.5.0

**To Reproduce**

1. Created storage class on v2.0.0 dev version as below

```
[root@oc-w3 fileset]# cat storageclassfileset.yaml
apiVersion: storage.k8s.io/v1
kind: StorageClass
metadata:
   name: ibm-spectrum-scale-csi-fileset
provisioner: spectrumscale.csi.ibm.com
parameters:
    clusterId: "6544305008369427178"
    volBackendFs: "gpfs0"
    filesetType: "dependent"
    gid: "9999"
    uid: "9999"
reclaimPolicy: Delete
```

Created the PVC using the above storage class 

Checked the Event log behaviour on 5.0.4.1 where log message says `The object "9999" specified for "user" does not exist`

```
Events:
  Type     Reason                Age               From                                                                                                 Message
  ----     ------                ----              ----                                                                                                 -------
  Warning  ProvisioningFailed    4s (x3 over 11s)  spectrumscale.csi.ibm.com_ibm-spectrum-scale-csi-provisioner-0_b9546b20-79f5-4811-871b-eb4d67619854  failed to provision volume with StorageClass "ibm-spectrum-scale-csi-fileset-test": rpc error: code = Internal desc = Unable to create fileset [pvc-d865ec68-ff8e-4b20-b427-d328d05fddb6] in FS [rgfps0]. Error [[EFSSP0010C CLI parser: The object "9999" specified for "user" does not exist.]]
  Normal   ExternalProvisioning  1s (x2 over 12s)  persistentvolume-controller                                                                          waiting for a volume to be created, either by external provisioner "spectrumscale.csi.ibm.com" or manually created by system administrator
  Normal   Provisioning          0s (x4 over 12s)  spectrumscale.csi.ibm.com_ibm-spectrum-scale-csi-provisioner-0_b9546b20-79f5-4811-871b-eb4d67619854  External provisioner is provisioning volume for claim "default/scale-fset-pvc"
```

While event log behaviour on 5.0.5.0 where log message says `Remote call completed with error [400 Bad Request]` is different. It should be consistent 

```
Events:
  Type     Reason                Age              From                                                                                                 Message
  ----     ------                ----             ----                                                                                                 -------
  Normal   ExternalProvisioning  6s (x2 over 6s)  persistentvolume-controller                                                                          waiting for a volume to be created, either by external provisioner "spectrumscale.csi.ibm.com" or manually created by system administrator
  Normal   Provisioning          3s (x3 over 6s)  spectrumscale.csi.ibm.com_ibm-spectrum-scale-csi-provisioner-0_b1491d4b-0c2f-4a2c-bada-d3c4d40ce7fe  External provisioner is provisioning volume for claim "ibm-spectrum-scale-csi-driver/scale-fset-pvc"
  Warning  ProvisioningFailed    2s (x3 over 6s)  spectrumscale.csi.ibm.com_ibm-spectrum-scale-csi-provisioner-0_b1491d4b-0c2f-4a2c-bada-d3c4d40ce7fe  failed to provision volume with StorageClass "ibm-spectrum-scale-csi-fileset": rpc error: code = Internal desc = Unable to create fileset [pvc-cdfdd62e-d640-420c-948f-30cf3d262363] in FS [gpfs0]. Error [Remote call completed with error [400 Bad Request]]

```

**Expected behavior**
Event logs are not consistent from 5.0.4.1 to 5.0.5.0

**Environment**
Please run the following an paste your output here:
``` bash
OCP 4.4.4
GPFS 5.0.5
```

**Comment by deeghuge** [comment]
This is due to changes in REST API in v5.0.5. Changes are done as part of Story#217007.

**Comment by Jainbrt** [comment]
@deeghuge do you think we can do this in 2.0.0 release ?

**Comment by deeghuge** [comment]
@Jainbrt : Fix was merged into v2.0.0
